csrc/python_frontend/fusion_state.cpp

```diff
@@ -68,6 +68,27 @@ std::ostream& operator<<(std::ostream& os, const State& state) {
   return os;
 }
 
+std::vector<Val*> getExtents(Fusion* fusion) {
```

Comment:
Moved `getExtents` out of `FusionState`, so it is accessible in `SegmentationState`.